packages/amplication-server/src/core/build/build.controller.ts

```diff
@@ -70,9 +69,11 @@ export class BuildController {
   @EventPattern(
     EnvironmentVariables.instance.get(Env.CREATE_PR_SUCCESS_TOPIC, true)
   )
-  async onPullRequestCreated(@Payload() message: KafkaMessage): Promise<void> {
+  async onPullRequestCreated(
+    @Payload() message: CreatePRSuccess
+  ): Promise<void> {
     try {
-      const args = plainToInstance(CreatePRSuccess, message.value);
+      const args = plainToInstance(CreatePRSuccess, message);
```

Comment:
is `plainToInstance` still needed? 
Nestjs will attempt to parse the message to JSON and the right type here